projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrService.java

```diff
@@ -555,6 +555,46 @@ public JSONArray getAnswer(
       }
    }
 
+   @POST
+   @Path(CoordConsts.SVC_RSC_GET_CONTAINER)
+   @Produces(MediaType.APPLICATION_JSON)
+   public JSONArray getContainer(
```

Comment:
getContainer() is perhaps not the right name for this function. Perhaps:
 -  getContainerInfo() if the plan down the line is to provide more information about the container (e.g., time of creation)
- doesContainerExists() is the intent is to simply let the clients check if the container with the provided name exists. (that is the purpose this function is currently serving, as no other information is provided.)